### PR TITLE
Merge train-124 into master

### PR DESCRIPTION
**Pull request by philbooth**
@mozilla/fxa-devs r?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+<a name="v1.124.1"></a>
+## v1.124.1 (2018-11-07)
+
+
+#### Bug Fixes
+
+* **mime:**  use base64 encoding instead of 8bit ([5d03a48d](5d03a48d))
```
Comment (vladikoff) [inline CHANGELOG.md]: did we miss out on the rest of the changes here?

Comment (philbooth) [reply CHANGELOG.md]: Yeah, the changelog script for this repo doesn't pick up `chore`. I plan to do something about it as part of #214.

Comment (vladikoff) [reply CHANGELOG.md]: yea clog doesn't like chores

Comment (philbooth) [reply CHANGELOG.md]: And `refactor` too I guess.

**Pull request merged**